File: pinyougou-manager-web/src/main/webapp/js/service/brandservice.js
//品牌服务层   Service
app.service("brandService", function ($http) {
    //查找列表
    this.findAll = function () {
        return $http.get('../brand/findAll.do');
    }

    //分页
    this.findPage = function (pageNum, pageSize) {
        return $http.get("../brand/findPage.do?pageNum=" + pageNum + "&pageSize=" + pageSize);
    }

    //添加
    this.addBrand = function (entity) {
        return $http.post('../brand/addBrand.do', entity);
    }

    //修改回显
    this.findOne = function (id) {
        return $http.post("../brand/findOne.do?id=" + id);
    }

    //修改数据
    this.update = function (entity) {
        return $http.post('../brand/update.do', entity);
    }

    //批量删除
    this.delete = function (id) {
        return $http.get("../brand/del.do?ids=" + id);
    }
    //条件查询
    this.search = function (pageNum,pageSize,seachEntity) {
        return $http.post("../brand/search.do?pageNum=" + pageNum + "&pageSize=" + pageSize,seachEntity);
    }

    //查找品牌下拉框
    this.selectOption=function(){
        return $http.get('../brand/selectOption.do');
    }

});

File: pinyougou-sellergoods-interface/src/main/java/com/pinyougou/sellergoods/service/BrandService.java
package com.pinyougou.sellergoods.service;

import com.pinyougou.pojo.TbBrand;
import vo.PageResult;


import java.util.List;
import java.util.Map;

public interface BrandService {

    public List<TbBrand> findAll();

    //分页 条件查找
    public PageResult findPage(int pageNum, int pageSize);

    //添加
    public void addBrand(TbBrand tbBrand);

    //修改
    public TbBrand findOne(long id);

    //开始修改
    void update(TbBrand tbBrand);

    //批量删除
    void delete(long[] id);

    //条件查询 分页
    PageResult Findsearch(TbBrand tbBrand, int pageNum, int pageSize);

    //查找品牌
    List<Map> selectOption();
}
